### PR TITLE
Add new sections to myths.md

### PR DESCRIPTION
**Pull request by Echolon**
Related https://yopad.eu/p/xsf-collection-privacy-security

Many thanks to everyone who committed their input. However, I don't know their names.

### CODE REVIEW

File: content/about/myths.md [hunk 1]
```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
 
 But even ignoring the high performance, specialist end, XML still holds its own. Consider [these findings](https://www.balisage.net/Proceedings/vol10/html/Lee01/BalisageVol10-Lee01.html), for example.
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure and does not implement modern technology.
```
Comment (jcbrand) [inline content/about/myths.md:27]: These are two separate hypotheses that are not necessarily related to one another.

I think this should be shortened to just `XMPP  is not secure` as this is what the subsequent text appears to be about.

File: content/about/myths.md [hunk 2]
```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
+
+The fact:
+> XMPP offers state-of-the-art end-to-end encryption, and a relatively high-degree of pseudonymity
+
+However, we do not ever claim that XMPP is secure. "Secure" is not an absolute. In order to decide whether XMPP is secure, you need to decide what you want to be secure against. No system that relies on servers is secure against for example a rogue administrator (end-to-end encryption can protect your message contents even against a rogue admin, but only if you verify keys appropriately). Instead, our specifications consistently remind developers of the risks of various choices. Plenty of XMPP extensions will certainly
```
Comment (jcbrand) [inline content/about/myths.md:32]: > However, we do not ever claim that XMPP is secure.

I would remove this sentence entirely, it's not necessary, is overly defensive, and looks as if you're admitting that XMPP is not secure.

File: content/about/myths.md [hunk 3]
```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
+
+Some messaging systems will expose less metadata. Some will ensure and enforce end to end encryption. Some will allow for federation, which inevitably increases the metadata involved. Others will sacrifice server-side message handling entirely, trading user-experience for reduced exposure. But the majority of the issues raised will affect any messaging system to some degree - sometimes better, and sometimes worse.
```
Comment (jcbrand) [inline content/about/myths.md:37]: > But the majority of the issues raised

What issues, raised by whom?

File: content/about/myths.md [hunk 4]
```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
+
+The community vision remains as it always has - that users are enabled to make their own choices about their client software, server software, and the people involved in both. We think XMPP enables exactly this.
+
+## XMPP is not privacy-friendly.
+
+The hypothesis:
+
+> XMPP is not privacy-friendly and exposes their users.
```
Comment (jcbrand) [inline content/about/myths.md:45]: Can be shortened to `XMPP is not privacy-friendly`.

The part `and exposes their users` is overly dramatic IMO and doesn't add anything to the first part of the sentence.

**Comment by Echolon** [comment]
@jcbrand do you approve my changes?

File: content/about/myths.md [hunk 5]
```diff
@@ -21,6 +21,36 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure.
+
+The fact:
+> XMPP offers state-of-the-art end-to-end encryption, and a relatively high-degree of pseudonymity
+
+"Secure" is not an absolute. In order to decide whether XMPP is secure, you need to decide what you want to be secure against. No system that relies on servers is secure against for example a rogue administrator (end-to-end encryption can protect your message contents even against a rogue admin, but only if you verify keys appropriately). Instead, our specifications consistently remind developers of the risks of various choices. Plenty of XMPP extensions will certainly increase the risk for both for the users and for the administrator - including allowing registration and password changing via the protocol.
```
Comment (jcbrand) [inline content/about/myths.md:32]: > Instead, our specifications consistently remind developers of the risks of various choices.

Might want to specifically mention the `Security Considerations` section.

For example:

Instead, XMPP specifications usually contain a "Security Considerations" section where developers are made aware of the risks of various choices.

File: content/about/myths.md [hunk 6]
```diff
@@ -21,6 +21,36 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
+
+Different deployment patterns radically alter the security, too. XMPP is designed by its nature to be highly flexible. This is why it is used by everything from computer games to military and nation states, and from telephone switchboards to the video game handhelds. Security that suits the military will rarely suit the average internet user, and vice-versa - XMPP strives to handle all of these, and it's certainly possible to configure a server to provide much worse security - or much better. That said, basic security is both encouraged and provided for; and this is most effective when using a server you trust. One way of trusting your server administrator is, of course, to be your own server administrator. This may sound obvious, but XMPP has from its outset ensured that this is always an option; even on low-budget.
```
Comment (jcbrand) [inline content/about/myths.md:34]: > from computer games to military and nation states

Sounds like you want to describe the scale here... so I'd rather write "from gamers to militaries and nation states".

> and from telephone switchboards to the video game handhelds

You can drop `the`.

File: content/about/myths.md [hunk 7]
```diff
@@ -21,6 +21,36 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
...
+
+> XMPP is not privacy-friendly.
+
+The fact:
+
+> As any other system, XMPP will interact with metadata and privacy must be enforced by the implementation and server adminstrators.
+
+XMPP enables in most aspects privacy-friendly implementations and usage. But there are no absolutes here. By using XMPP - or any other system - you give access to your metadata to some degree. XMPP is certainly not the system with the least metadata available to an observer, especially an observer with access to your server. But no single server administrator can possibly see all the metadata you provide via your interactions.
+
+By federation XMPP provides the user with the ability to trust just one server administrator or even split trust over several servers using multiple accounts, and remote servers will only see metadata that concerns them.
```
Comment (jcbrand) [inline content/about/myths.md:52]: `By federation` should be `By allowing federation`. Also, I think we can mention self-hosting here.

For example:

By allowing federation XMPP provides the user with the ability to self-host their own server, to choose which server administrator to trust, or even to split trust over several servers by using multiple accounts. Remote servers from contacts will only see the relevant metadata that concerns them.

**Review by jcbrand — COMMENTED** [review]
Sorry for the delay @Echolon, looks good. I have a few more suggestions if you're up for it.

**Comment by Echolon** [comment]
@jcbrand I added all changes. Many thanks!

**Review by jcbrand — APPROVED** [review]

**Pull request merged**